halconfig/images.yml

```diff
@@ -13,26 +13,36 @@ aws:
         instanceType: t2.micro
         sourceAmi: ami-d4aed0bc
         sshUserName: ubuntu
+        spotPrice: auto
+        spotPriceAutoProductType: Linux/UNIX (Amazon VPC)
```

Comment:
This is `spotPriceAutoProduct` everywhere else.

Reply:
Fixed. Thanks for the catch.